packages/html-reporter/src/filter.ts

```diff
@@ -114,6 +120,7 @@ export class Filter {
         line: String(test.location.line),
         column: String(test.location.column),
         labels: test.tags.map(tag => tag.toLowerCase()),
+        annotations: test.annotations.map(a => a.type.toLowerCase() + ':' + a.description?.toLocaleLowerCase())
```

Comment:
To make sure you only use : once in the filter.

```suggestion
        annotations: test.annotations.map(a => a.type.toLowerCase() + '=' + a.description?.toLocaleLowerCase())
```